src/neo/Network/P2P/TaskManager.cs

```diff
@@ -18,6 +18,7 @@ public class Register { public VersionPayload Version; }
         public class Update { public uint LastBlockIndex; }
         public class NewTasks { public InvPayload Payload; }
         public class RestartTasks { public InvPayload Payload; }
+        private class Counter { public int Value; }
```

Comment:
Use `Tuple<T>`? But I think this optimization is of little significance.

Reply:
Tupple is immutable

Reply:
But this is weird. Can we just use `Dictionary<UInt256, int>`?

Reply:
The problem with uint as value if that for change the value we need to search again in the dictionary

Reply:
If the key does not exist in the dictionary, you will have to create a new object (`Counter`), which is most of the time. So it is hard to say whether there is a performance improvement. And even if there is an improvement, I don't expect much improvement. And this reduces the code readability.

Reply:
So do you mind changing it back? @shargon